bids/variables/io.py

```diff
@@ -6,6 +6,7 @@
 from .entities import NodeIndex
 from .variables import SparseRunVariable, DenseRunVariable, SimpleVariable
 import warnings
+from bids.config import _settings
```

Comment:
Let's try to respect the encapsulation and use `set_option` and `get_option` calls if possible, instead of directly accessing `_settings`.